src/plugins/dashboard/public/application/listing/dashboard_unsaved_listing.tsx

```diff
@@ -59,6 +60,7 @@ const DashboardUnsavedItem = ({
             color="primary"
             onClick={onOpenClick}
             data-test-subj={`edit-unsaved-${dashboard?.id ?? DASHBOARD_PANELS_UNSAVED_ID}`}
+            aria-label={`${dashboardUnsavedListingStrings.getEditTitle()} ${title}`}
```

Comment:
Nit on i18n (here and below):

Because we don't know how other languages build there sentences, we should create a new string for this so that translators get a chance to arrange it if they need to for their language

Reply:
Makes sense, fixed this.